<ui version="4.0" >
 <class>ClientHierarchyView</class>
 <widget class="QWidget" name="ClientHierarchyView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="horizontalSplitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="treeView" >
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSplitter" name="verticalSplitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QVTKWidget" native="1" name="hierarchicalGraphView" />
      <widget class="QVTKWidget" native="1" name="graphView" />
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
